src/esp/sensor/FisheyeSensor.cpp

```diff
@@ -52,8 +52,7 @@ Magnum::Vector2 computePrincipalPointOffset(const FisheyeSensorSpec& spec) {
   if (bool(spec.principalPointOffset)) {
     return *spec.principalPointOffset;
   }
-  auto res = spec.resolution.cast<float>();
-  return Mn::Vector2(res[0], res[1]) * 0.5f;
+  return Mn::Vector2(spec.resolution[0], spec.resolution[1]) * 0.5f;
```

Comment:
The `spec.resolution` is a Magnum `Vector2i` now, right? So

```suggestion
  return Mn::Vector2{spec.resolution} * 0.5f;
```